outputs/jira.go

```diff
@@ -285,6 +273,114 @@ func createMetaProject(c *jira.Client, project string) (*jira.MetaProject, error
 	return metaProject, nil
 }
 
+func createIssueType(ctx *JiraAPI, metaProject *jira.MetaProject) error {
```

Comment:
1. verb in function shouldn't be `create` 
2. Check if it's possible to avoid side effects 
3. search loop is implemented twice
4. "don't" -> "doesn't"

Reply:
Done